common/changes/@microsoft/rush/sennyeya-shard-schema-updates_2024-09-13-22-46.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "Remove sharding.shardOperationSettings. Use a new operationSettings object with a name of ':shard'",
```

Comment:
```suggestion
      "comment": "(BREAKING CHANGE) Remove the `sharding.shardOperationSettings` property in the project `config/rush-project.json` in favor of an `operationSettings` entry for an operation with a suffix of `:shard`.",
```

Reply:
Updated to a deprecation notice instead.